### ISSUE TITLE
Focus appearance in microsite header

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_microsites_base#282]
With the default microsite base theme - no overrides - the header background is black and the text white. The link:focus gives the link a yellow background but doesn't change the text colour. 

![image](https://github.com/user-attachments/assets/36d1bfe6-c05a-4fb9-8232-546dedd7076f)

Can we ensure the text is still legible in focus please. 

Also, please could we have a visible focus appearance on the logo in the header, if there is one.

**Comment by markconroy** [comment]
To do:

- [x] Can we ensure the text is still legible in focus please.
- [x] Also, please could we have a visible focus appearance on the logo in the header, if there is one.
